src/openvswitch.c

```diff
@@ -29,6 +29,42 @@
 #include "util.h"
 #include "util-internal.h"
 
+#ifndef PREFIX 
+#define PREFIX "/usr/"
+#endif
+
+#ifndef BINDIR
+#define BINDIR PREFIX"bin"
+#endif
+
+#ifndef SNAPBINDIR
+#define SNAPBINDIR "/snap/bin"
+#endif
+
+#define OPENVSWITCH_OVS_VSCTL "ovs-vsctl"
+
+static GString *netplan_openvswitch_ovs_vsctl_path = NULL;
+
+void netplan_openvswitch_vsctl__(void)
```

Comment:
suggestion: Let's use a more telling name here, e.g.: `void _prepare_netplan_openvswitch_ovs_vsctl_path__(void)`

We might also add an `g_assert(netplan_openvswitch_ovs_vsctl_path == NULL)` at the top of the function. It case somebody tries to call it outside of `netplan_openvswitch_ovs_vsctl_` in the future.